extension/chrome/elements/compose-modules/compose-sender-module.ts

```diff
@@ -67,5 +67,9 @@ export class ComposeSenderModule extends ViewModule<ComposeView> {
   private actionInputFromChangeHandler = async () => {
     await this.view.recipientsModule.reEvaluateRecipients(this.view.recipientsModule.getValidRecipients());
     this.view.footerModule.onFooterUpdated(await this.view.footerModule.getFooterFromStorage(this.view.senderModule.getSender()));
+    const signaturePattern = /--\s*<br\s*\/?>[\s\S]*$/i;
```

Comment:
this regex works well for new messages, but when you compose reply message and switch send mail address - it removes older messages content too.

need to update this regex to leave older messages text (starting from `On ... wrote:...`)

Reply:
Nice catch :+1: 
Implemented a fix and pushed